io.gitlab.celleron56.libretile.yaml

```diff
@@ -0,0 +1,92 @@
+app-id: io.gitlab.celleron56.libretile
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: /app/files/start.sh
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=all
+  - --share=network
+
+
+modules:
+  - name: libretile
+    buildsystem: simple
+    
+    build-commands:
+       - mkdir /app/files
+       - mv * /app/files
```

Comment:
instead you should just copy what's needed. Ideally upstream would include a installation rule like any buildsystem provide.

Also `/app/files` is not really FSB compliant.

Reply:
well the issue is that  mostly  everything is needed   (and     things which are not needed are documentation ) (which directory should i use instead )

Reply:
like the manual.odp file? the libretile.desktop ?

Reply:
manual.odp is the user manual  (which i can probably put a delete step for ) and the libretile desktop file is needed  (it is moved to the /share/applications folder )

Reply:
> instead you should just copy what's needed. Ideally upstream would include a installation rule like any buildsystem provide.
> 
> Also `/app/files` is not really FSB compliant.

the flatpak documentation does not state anything about fsb

https://docs.flatpak.org/en/latest/search.html?q=fsb&check_keywords=yes&area=default

so   i cannot fix that since i do not have  documentation about FSB and therefore any possible fix i might attempt 
might be uncompliant as well

Reply:
Sorry I mean "LSB FHS" https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard

`/app` is the prefix, that mean executables are in `/app/bin`, shared data in `/app/share`, libraries in `/app/lib`. etc. `/app/bin` is in the `PATH`.

While it's generally acceptable to have a `/app/MyApp`, `files` isn't your app name. 

You sould really have an install procedure upstream and use that.